tox.ini

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+[tox]
+skipsdist=True
+skip_missing_interpreters = True
+envlist = lint, static, unit
+
+[vars]
+src_path = {toxinidir}/ops/
+tst_path = {toxinidir}/test/
+lib_path = {toxinidir}/test/charms/test_main/lib
```

Comment:
I think we can drop this -- this was added specifically to facilitate running mypy on charm code + charm library code. I'm not sure it'll be useful here.

Reply:
(To be clear, I just mean the lib_path variable)

Reply:
+1